app/src/test/kotlin/com/inout/apiserver/application/study/CreateStudyApplicationTest.kt

```diff
@@ -117,7 +129,7 @@ class CreateStudyApplicationTest {
             )
         val word = createWords(1).first()
         val userId = 1L
-        val study = createStudyWithWord(word, userId)
+        val study = createStudy(word, userId)
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/in-out-team/api-server/pull/12#issuecomment-2364605989>#73b7d8</a><span> - 09/20/2024, 09:08 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The test case does not verify the FSRS-related properties of the Study object after changing to the createStudy function.
</div>
<div id="fix">
<b>Fix:</b> Update the test case to assert the FSRS-related properties of the returned Study object.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -140,3 +140,11 @@
      assertEquals(study, sut.study)
      assertEquals(word, sut.word)
 +    // Assert FSRS properties
 +    assertEquals(FsrsCardState.NEW, sut.study.state)
 +    assertEquals(Card.createEmptyCard().due, sut.study.due)
 +    assertEquals(Card.createEmptyCard().stability, sut.study.stability)
 +    assertEquals(Card.createEmptyCard().difficulty, sut.study.difficulty)
 +    assertEquals(Card.createEmptyCard().elapsedDays, sut.study.elapsedDays)
 +    assertEquals(Card.createEmptyCard().scheduledDays, sut.study.scheduledDays)
 +    assertEquals(Card.createEmptyCard().reps, sut.study.reps)
 +    assertEquals(Card.createEmptyCard().lapses, sut.study.lapses)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged